pages/wizard/openshift/nodes.py

```diff
@@ -0,0 +1,124 @@
+from selenium.webdriver.common.by import By
+from pages.base import Base
+
+class Nodes(Base):
+    _page_title = "QuickStart Cloud Installer"
+    _rhv_radio_loc = (By.XPATH, "//input[@value='RHEV']")
+    _master_node_count_loc = (By.XPATH, "//span[@data-qci='master-1']")
+    _worker_custom_link_loc = (By.XPATH, "//a[@data-qci='show-custom-worker-nodes']")
+    _storage_custom_link_loc = (By.XPATH, "//a[@data-qci='show-custom-storage-size']")
+    _custom_edit_loc = (By.XPATH, "//button[contains(., 'Custom Edit')]")
+    _finish_editing_loc = (By.XPATH, "//button[contains(., 'Finish Editing')]")
+    _master_vcpu_loc = (By.ID, "master-v_cpu")
+    _master_ram_loc = (By.ID, "master-ram")
+    _master_disk_loc = (By.ID, "master-disk")
+    _worker_vcpu_loc = (By.ID, "worker-v_cpu")
+    _worker_ram_loc = (By.ID, "worker-ram")
+    _worker_disk_loc = (By.ID, "worker-disk")
+
+    @property
+    def rhv_radio_button(self):
+        return self.selenium.find_element(*self._rhv_radio_loc)
+
+    @property
+    def master_node_count(self):
+        return self.selenium.find_element(*self._master_node_count_loc)
+
+    def worker_node_count(self, number):
```

Comment:
This is an experiment. I wanted to make locator that would change according to user input instead of just listing all the locators here. In order to make it work I had to un-@property it though.